addons/mail/security/mail_security.xml

```diff
@@ -162,6 +162,36 @@
             <field name="perm_unlink" eval="False"/>
         </record>
 
+        <record id="ir_rule_discuss_call_history_read_all" model="ir.rule">
+            <field name="name">discuss.call.history: read call history of accessible channels</field>
```

Comment:
Don't forget override in live chat to give all rights to live chat user for live chat sessions.